gradle/libs.versions.toml

```diff
@@ -23,6 +25,7 @@ androidx-activity-compose = { module = "androidx.activity:activity-compose", ver
 compose-bom = { module = "androidx.compose:compose-bom", version.ref = "composeBom" }
 compose-ui = { module = "androidx.compose.ui:ui" }
 compose-ui-graphics = { module = "androidx.compose.ui:ui-graphics" }
+compose-foundation = { module = "androidx.compose.foundation:foundation" }
```

Comment:
Majority of compose dependencies will be used in all submodules -> worth to create a bundle in the version catalog and reuse it?

Reply:
Added :)